js/ngDialog.js

```diff
@@ -7,7 +7,7 @@
 (function (root, factory) {
     if (typeof module !== 'undefined' && module.exports) {
         // CommonJS
-        module.exports = factory(require('angular'));
+        module.exports = require('angular');
```

Comment:
@voronianski Yes I do. Also, ngDialog does not export anything so the module is loaded under a scoped copy of angular.

Right now I just have the module loading code removed and everything is working fine.